liblangutil/SourceLocation.cpp

```diff
@@ -27,6 +27,34 @@ using namespace solidity;
 using namespace solidity::langutil;
 using namespace std;
 
+string SourceLocation::singleLineSnippet(StringMap const& _sourceCodes) const
+{
+	if (!this->hasText() || _sourceCodes.empty())
+		return {};
+
+	auto it = _sourceCodes.find(*this->sourceName);
+	if (it == _sourceCodes.end())
+		return {};
+
+	return singleLineSnippet(it->second);
+}
+
+string SourceLocation::singleLineSnippet(string const& _sourceCode) const
+{
+	if (!this->hasText())
```

Comment:
We usually never use explicit `this`...

Reply:
Oh, true. I did not notice that `this` is pointless here when moving this :)

Reply:
No, wait. I replaced them myself :) Sorry, not sure how I did not notice this earlier :)

Anyway, removed.